src/main/resources/org/sagebionetworks/web/client/widget/entity/RejectReasonViewImpl.ui.xml

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+             xmlns:g="urn:import:com.google.gwt.user.client.ui" xmlns:c="urn:import:com.google.gwt.user.cellview.client"
+             xmlns:b="urn:import:org.gwtbootstrap3.client.ui" xmlns:t="urn:import:org.sagebionetworks.web.client.widget.table.v2"
+             xmlns:a="urn:import:org.sagebionetworks.web.client.widget.table.v2"
+             xmlns:bg="urn:import:org.gwtbootstrap3.client.ui.gwt" xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html">
+
+    <b:Modal ui:field="modal" title="Response" closable="true"
+             dataBackdrop="STATIC" dataKeyboard="true">
+        <b:ModalBody>
+            <bh:Div>
+                <b:FormGroup>
+                    <b:CheckBox ui:field="optionOne"   text="Take and pass the Synapse Certification quiz. "/>
```

Comment:
Please use the constants to set the CheckBox text (so they don't exist in 2 places in the code base, which might get out of sync)